Gds/src/fprime_gds/executables/fprime_cli.py

```diff
@@ -79,7 +81,8 @@ def add_search_arguments(parser: argparse.ArgumentParser, command_name: str):
         "--list",
         dest="is_printing_list",
         action="store_true",
-        help="list all possible %s types the current F Prime instance could produce, based on the %s dictionary, sorted by %s type ID"
+        help="list all possible %s types the current F Prime instance could produce,"
+             "based on the %s dictionary, sorted by %s type ID"
```

Comment:
Same as above. I prefer to not separate the strings into two quotes on separate line. Might be better to revert it back to what they were.